src/app/src/components/EmbeddedMapConfig.jsx

```diff
@@ -166,16 +168,28 @@ function EmbeddedMapConfig({
                 </Grid>
                 <Grid item xs={12} md={7} lg={6}>
                     {user.embed_level === 3 ? (
-                        <EmbeddedMapFieldsConfig
-                            fields={fields}
-                            setFields={setFields}
-                            preferContributorName={preferContributorName}
-                            setPreferContributorName={updateEmbedConfig(
-                                'preferContributorName',
-                            )}
-                            errors={errors}
-                        />
+                        <>
+                            <EmbeddedMapFieldsConfig
+                                fields={fields}
+                                setFields={setFields}
+                                preferContributorName={preferContributorName}
+                                setPreferContributorName={updateEmbedConfig(
+                                    'preferContributorName',
+                                )}
+                                errors={errors}
+                            />
+                            <EmbeddedMapSearchLabelConfig
```

Comment:
Seems that `<EmbeddedMapSearchLabelConfig>` might be better off as a child of `<EmbeddedMapFieldsConfig>`, for a couple of reasons:
1. Semantically / structurally, the search label config should be a subsection of the "Fields" section. It looks that way on the rendered page, but in the DOM it's a sibling, not a child.
2. It is dependent on the fields config checkboxes.